src/main/java/net/geforcemods/securitycraft/SCEventHandler.java

```diff
@@ -223,7 +224,10 @@ public void onRightClickBlock(RightClickBlock event){
 						PlayerUtils.sendMessageToPlayer(event.getEntityPlayer(), ClientUtils.localize("item.securitycraft:universalBlockRemover.name"), ClientUtils.localize("messages.securitycraft:notOwned").replace("#", ((IOwnable) tileEntity).getOwner().getName()), TextFormatting.RED);
 						return;
 					}
-
+					if(block == SCContent.keypadChest) {
+						BlockKeypadChest chest = (BlockKeypadChest) block;
+						chest.breakBlock(world,event.getPos(),chest.getBlockState().getBaseState());
+					}
```

Comment:
1. Please move all of this to the start of the else block in line 252
2. Use `world.getBlockState(event.getPos())` instead of `chest.getBlockState().getBaseState()`
3. Add a space after each `,` and remove the `chest` variable

Reply:
done, thanks for your time.